src/main.tsx

```diff
@@ -0,0 +1,10 @@
+import { StrictMode } from "react";
+import { createRoot } from "react-dom/client";
+import "./index.css";
+import EggCookingHelper from "./eggCookingHelper.tsx";
```

Comment:
Not a big thing, but it's unusual to rename the app.tsx in a project. Other devs reading this and seeing it's a React project would typically search for an app.tsx file as the kind of "main entry point".